### ISSUE TITLE
notebook check is very slow

### ISSUE THREAD

**Issue by richrines1** [Infleqtion/client-superstaq#889]
the CI notebook check generally takes 10-15 minutes to complete (compared to 1-3 minutes for each of the other checks)

ideally we should speed up whatever steps are making it so slow, but in the interim maybe it would make sense to split it up into separate tests for `*_qss.ipynb`, `*_css.ipynb`, and "other" so at least the run in parallel?

**Comment by stephanielee9** [comment]
~7 minutes now
